src/transactions/transactions.service.ts

```diff
@@ -87,7 +87,7 @@ export class TransactionsService {
         );
       }
     } catch (error) {
-      handleSdkError(error);
+      handleSdkError(error as Error);
```

Comment:
I think `handleSdkError` should update its signature to be 

```
handleSdkError(err: unknown): never
```

then at the bottom of the function to:

```
  if (err instanceof Error) {
    throw new InternalServerErrorException(err.message);
  }

  throw err; // maybe throw new InternalServerErrorException('An unexpected error occurred') instead
````